content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-have the possibility to specify the unit of measure in which you manage
-your stock and the one which is used when purchasing the product to your
-supplier.
+Units of measure specify the unit used to handle a product. In Odoo, you have the possibility to
+specify the unit of measure in which you manage your stock and the one which is used when
+purchasing the product to your supplier.
 
-.. image:: usage/usage_01.png
-    :align: center
+.. image:: usage/uom-handling-vs-purchase.png
+   :align: center
+   :alt: Specify unit of measure for handling a product vs. when it's purchased
 
-The *conversion* between the different units of measure is done
-automatically. The only condition is that all the units have to be in
-the *same category* (Unit, Weight, Volume, Length,...)
+The *conversion* between the different units of measure is done automatically. The only condition
+is that all the units have to be in the *same category* (Unit, Weight, Volume, Length,...)
 
-For example, if I have the following reordering rule for the egg and I
-run the scheduler, the quantity added in the automatically generated
-purchase order will be in dozens but what will enter the stock will be
-units.
+For example, if I have the following reordering rule for the egg and I run the scheduler, the
+quantity added in the automatically generated purchase order will be in dozens but what will enter
+the stock will be units.
 
-.. image:: usage/usage_02.png
-    :align: center
+.. image:: usage/units-of-measure-conversion.png
+   :align: center
+   :alt: Units of Measure conversion happens automatically with quantity multiplier field
 
-.. image:: usage/usage_03.png
-    :align: center
+.. image:: usage/uom-conversion-on-purchase-order.png
+   :align: center
+   :alt: Units of Measure converts to appropriate quantity on generated purchase orders
 
-.. image:: usage/usage_04.png
-    :align: center
+.. image:: usage/uom-conversion-by-category.png
+   :align: center
+   :alt: Units of Measure conversions happen automatically as long as units are in the same category.
 
 Packages
 ========
 
-The package is the physical container in which you put one or several
-products from a picking. For example, when you deliver a product, you
-can decide to separate the quantity into two different packages. It then
-allows you to have a report with the quantity of products for each
-package.
+The package is the physical container in which you put one or several products from a picking. For
+example, when you deliver a product, you can decide to separate the quantity into two different
+packages. It then allows you to have a report with the quantity of products for each package.
 
-To separate a delivery into different packages you will have to set the 
-done quantity to the desired package quantity then click on "PUT IN 
-PACK", do this for each package.
+To separate a delivery into different packages you will have to set the done quantity to the
```

Comment:
Replace the one sentence on Lines 42-43 `To separate a delivery...do this for each package` with:

`On a delivery order, separate the products into different packages by setting the :guilabel:'Done' quantity to the desired quantity in the first package. Then, click :guilabel:'Put in Pack' to record the first package. Repeat for each package.`

(^ single quotation marks = backticks in my suggested change)